### ISSUE TITLE
Install fails when repo does not yet have an `.env.vault` file

### ISSUE THREAD

**Issue by motdotla** [davidcochrum/dotenv-vault-laravel#2]
When my laravel repo has no `.env.vault` file and I run the following command:

```
composer require davidcochrum/dotenv-vault-laravel
```

I receive the following error:

```
$yourapp: composer require davidcochrum/dotenv-vault-laravel

./composer.json has been updated
Running composer update davidcochrum/dotenv-vault-laravel
Loading composer repositories with package information
Updating dependencies
Nothing to modify in lock file
Writing lock file
Installing dependencies from lock file (including require-dev)
Nothing to install, update or remove
Generating optimized autoload files
> Illuminate\Foundation\ComposerScripts::postAutoloadDump
> @php artisan package:discover --ansi

   Dotenv\Exception\InvalidPathException 

  Unable to read any of the environment file(s) at [/Users/scottmotte/Code/dotenv-org/temp/yourapp/.env.vault].

  at vendor/vlucas/phpdotenv/src/Store/FileStore.php:68
     64▕         if (\count($contents) > 0) {
     65▕             return \implode("\n", $contents);
     66▕         }
     67▕ 
  ➜  68▕         throw new InvalidPathException(
     69▕             \sprintf('Unable to read any of the environment file(s) at [%s].', \implode(', ', $this->filePaths))
     70▕         );
     71▕     }
     72▕ }

      +10 vendor frames 

  11  artisan:35
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))

Script @php artisan package:discover --ansi handling the post-autoload-dump event returned with error code 1
```

It appears the install script expects there to a `.env.vault` file. Many times the user will not yet have a `.env.vault` file.

**Comment by davidcochrum** [comment]
Resolved with [v1.1.1](https://github.com/davidcochrum/dotenv-vault-laravel/releases/tag/v1.1.1)

**Comment by motdotla** [comment]
<img width="515" alt="Screenshot 2023-11-07 at 8 46 03 PM" src="https://github.com/davidcochrum/dotenv-vault-laravel/assets/3848/8258d13c-d3e5-4531-b245-1e539b1930a1">

---

https://www.dotenv.org/docs/frameworks/laravel/laravel-forge
